src/components/backend-ai-import-view.ts

```diff
@@ -581,6 +581,18 @@ export default class BackendAIImport extends BackendAIPage {
     }
   }
 
+  urlTextfieldChanged(e, buttonIdValue) {
+    const button = this.shadowRoot.querySelector(`#${buttonIdValue}`);
+    if (e.target.value !== '' && e.currentTarget.checkValidity()) {
+      button.removeAttribute('disabled');
+    } else {
+      this.notification.text = _text('import.WrongURLType');
+      button.setAttribute('disabled', 'true');
+      this.importMessage = this.notification.text;
```

Comment:
I guess we should split the importMessage to import notebook and import GitHub repo, since when either the value of the input field of Import Notebook or Import From GitHub is invalid, the error message is shown in both areas.
Here are the screenshots that refer to the issue that I mentioned above.

### Screnshots
![GitHub_Repo mov](https://user-images.githubusercontent.com/46954439/185544627-5f1696e5-57f6-4382-97bd-293e71a75ee1.gif)
<img width="673" alt="ImportNotebookRepo" src="https://user-images.githubusercontent.com/46954439/185544651-b74117dc-2417-42f0-8396-9358d205f849.png">